go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	github.com/jteeuwen/go-bindata v3.0.8-0.20151023091102-a0ff2567cfb7+incompatible
 	github.com/miekg/dns v1.1.15
 	github.com/modern-go/reflect2 v1.0.1
+	github.com/onsi/ginkgo v1.11.0
```

Comment:
After adding ginkgo, you should run `go mod vendor` so that it's included in the vendor directory.

Reply:
I did run "go mod tidy" and "go mod vendor", but the results are not as expected. Is there any other way?